README.md

```diff
@@ -70,8 +70,8 @@
 
 ## 🚀 Live Demo <a name="live-demo"></a>
 
-- coming soon after aprroved ....
-- coming soon after approved ....
+- coming soon ....
+- [Watch](https://www.loom.com/share/7a8ade753ca644c59e1e6ee241187278)
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
```

Comment:
- [ ] Tests are failing in your project
![image](https://user-images.githubusercontent.com/58504655/227194474-fc071e75-f6e2-423d-8ba2-7885eae578fc.png)